samples/knative/quickstart/platformsetup.sh

```diff
@@ -30,6 +30,7 @@ if [[ ! " ${NETWORK_LAYERS[@]} " =~ " ${1} " ]]; then
 fi
 
 # Step 1: Export correct tags for install artifacts
+export GIT_ACCOUNT="${GIT_ACCOUNT:iter8-tools}"
```

Comment:
Why is there a minus in lines 34 and 35 but none in 33?

Reply:
my mistake. There should be a "-"; see https://stackoverflow.com/a/48829326/3482067